### DISCUSSION TITLE
Pluralsight & Stack Overflow

### DISCUSSION THREAD

**Discussion by felipechalreo** [frontendbr/forum#931] [Dicas]
Eu recebi um email hoje falando deste Pluralsight IQ
https://www.pluralsight.com/partners/stack-overflow/iq

Parece ser interessante. Nem que seja para testar as próprias habilidades...

**Comment by willianjusten** [comment]
Me lembro de quando a Pluralsight começou a adicionar mini-testes para algumas áreas e aí depois ia testando e analisando se o conhecimento melhorou.

Me parece que esse IQ é uma extensão disso, bem bacana, depois vou testar =D

**Comment by carloshenriqueribeiro** [comment]
Amo pluralsight 😍

**Comment by rubenmarcus** [comment]
legal e o que acham do Hacker Rank? Além dele, e esse do PluralSight, Front-end Challenges, Upwork, aqueles de JS em 100 dias, quais sites mais vocês conhecem pra treinar conhecimento??

Uma vez vi uma palestra no Frontend Meetup aqui em SP, o palestrante falou de um site que te ajuda a 'testar' ou 'ensinar' Solid , TDD , etc e outras coisas de programação ,que hoje é meu maior gargalo. Se vocês conhecerem sites que ajudam mais na parte teórica e menos na de sintaxe, ia me ajudar muito..

**Comment by weltonsanches** [comment]
> Além dele, e esse do PluralSight, Front-end Challenges, Upwork, aqueles de JS em 100 dias, quais sites mais vocês conhecem pra treinar conhecimento??

@rubenmarcus [este repositório](https://github.com/alinebastos/dev-practice) da @alinebastos tem vários links com desafios legais.

**Comment by alinebastos** [comment]
Pior que eu fiz esse teste do Pluralsight de JS em setembro passado, e agora não consigo refazer pra ver se meus resultados melhoram! =/
